src/components/stake/StakeValidatorListItem.vue

```diff
@@ -0,0 +1,228 @@
+        margin-bottom: -.625rem;
+        box-shadow: none;
+        // display: inline-block;
+    }
+
+    input::-moz-focus-inner {
+        border: 0;
+    }
+
+    &:hover {
+        background-color: #f2f2f4;
+
+    }
+
+    .validator-item-wrapper {
+        display: flex;
+        width: 100%;
+        height: 100%;
+        flex-direction: row;
+        // padding: 1.75rem 2.5rem;
+        .validator-left {
+            margin-top: 2.25rem;
+            margin-left: 2.5rem;
+            margin-bottom: -2rem;
+            svg {
+                width: 5rem;
+                height: 5rem;
+                border-radius: 2.5rem; // temporary, this will be req just for the mock data I guess
+            }
+        }
+        .validator-item-mid {
+            display: flex;
+            flex-direction: column;
+            width: 100%;
+            padding-top: 2.25rem;
+            padding-left: 2.25rem;
+        }
+        .validator-item-inner-row {
+            display: flex;
+            flex-direction: row;
+            height: 3rem;
+            line-height: 3rem;
+            &.validator-label {
+                padding-top: 0.75rem;
+                font-weight: 600;
+                font-size: 2rem;
+                line-height: 100%;
+                color: var(--nimiq-blue);
+            }
+            &.validator-trust {
+            }
+        }
+        .validator-item-right {
+            height: 3rem;
+            min-width: 13rem;
+            line-height: 2rem;
+            margin-top: 3.75rem;
+            padding-top: 0.25rem;
+
+            border: 2px solid var(--nimiq-green);
+            border-radius: 1.5rem;
+            justify-content: flex-end;
+            white-space: nowrap;
+            text-align: center;
+            color: var(--nimiq-green);
+            font-size: 2rem;
+            font-weight: bold;
+
+            &.validator-warning {
+                border: 2px solid var(--nimiq-orange);
+                color: var(--nimiq-orange);
+            }
+        }
+        .validator-trust {
+            font-family: Mulish;
+            font-style: normal;
+            font-weight: 600;
+            font-size: 1.75rem;
+            color: rgba(31, 35, 72, 0.5);
+        }
+        .validator-payout {
+            white-space: nowrap;
+            padding-left: 1rem;
+            padding-top: 0.5rem;
+            // font-family: Mulish;
+            font-style: normal;
+            font-weight: 400;
+            font-size: 1.75rem;
+            line-height: 100%;
+            color: var(--nimiq-blue);
+
+            opacity: 0.5;
+        }
+    }
+    .red-star {
+        path {
+            fill: var(--nimiq-red);
+        }
+    }
+    .star {
+        margin-right: 0.125rem;
+        svg {
+            width: 1.5rem;
+            height: 1.5rem;
+        }
+    }
+    .gold-text {
+        color: var(--nimiq-gold);
+    }
+    .red-text {
+        color: var(--nimiq-red);
+    }
+}
```

Comment:
There are also a few unnecessary style declarations here. The font-family is global, does not need to be restated. The font-size should be set on a parent, instead of on each child individually. We also have utility classes `flex-row` and `flex-column` to replace `display: flex; flex-direction <row|column>;`.